source/return_api.rst

```diff
@@ -26,6 +26,9 @@ subtotal_delta                 No          Must be provided if the original tran
 total_value_delta              Yes         The delta value of transaction's grand total after returned
 payments                       No          Must be provided if original transaction has payments.
                                            Payments are list of :ref:`payment objects <Payment Object>`
+stamps_to_add                  No          Stamps to be added manually. Can't be sent alongside stamps_to_deduct.
+stamps_to_deduct               No          Stamps to be deducted manually, if this field is present, automatic stamps calculation will be overriden by this.
```

Comment:
`If specified, this overrides the number of Stamps that will be deducted from this return.`